chef/cookbooks/crowbar/templates/default/sysconfig.crowbar.erb

```diff
@@ -0,0 +1,43 @@
+## Path:            Crowbar
+## Description:     Mode of operation
+## Type:            string(production,development,test)
+## Default:         production
+## ServiceRestart:  crowbar
+# Crowbar can run in production, development or test mode. Normally, this
+# determines which database is used, but it may also have other
+# implications.
+CROWBAR_ENV="production"
+
+## Path:            Crowbar
+## Description:     Maximum number of threads
+## Type:            integer
+## Default:         1
+## ServiceRestart:  crowbar
+# Sets the maximum number of threads used by the web server.
+CROWBAR_THREADS="1"
+
+## Path:            Crowbar
+## Description:     Maximum number of worker processes
+## Type:            integer
+## Default:         2
+## ServiceRestart:  crowbar
+# Sets the maximum number of separate worker processes spawned by the
+# web server.
+CROWBAR_WORKERS="5"
```

Comment:
Please use the same defaults for all env vars in the service file, in the comments here, in the env vars here and in the puma.rb file.